telegram/chat.py

```diff
@@ -212,3 +212,107 @@ def unban_member(self, *args, **kwargs):
 
         """
         return self.bot.unban_chat_member(self.id, *args, **kwargs)
+
+    def send_message(self, *args, **kwargs):
+        """Shortcut for::
+
+            bot.send_message(Chat.chat_id, *args, **kwargs)
+
+        Where Chat is the current instance.
+
+        Returns:
+            :class:`telegram.Message`: On success, instance representing the message posted.
+
+        """
+        return self.bot.send_message(chat_id=self.id, *args, **kwargs)
```

Comment:
That was the exact point I was gonna make xD
Does it matter what error it throws? 
It seems like a lot of code for some superficial error that's unlikely to ever happen (as far as I understand it, it will never happen if users don't do stupid stuff, right?)

Reply:
Explicit errors are much easier to debug, especially in async systems, rather than deciphering a stack trace.
If you feel that the code is good as it is I won't object. I simply suggest to reconsider.

Reply:
i will leave this as is.